### PR TITLE
Moving FAQ - Adjusting identifiers for UCSC sourced data used with other sourced data

### PR DESCRIPTION
**Pull request by Melkeb**
This FAQ is part of [Mismatched Chromosome identifiers (and how to avoid them)](https://galaxyproject.org/support/chrom-identifiers/)

### CODE REVIEW

File: faqs/galaxy/datasets_adjust_identifiers_UCSC.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Adjusting identifiers for UCSC sourced data used with other sourced data
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+- A GTF formatted dataset (potentially a "reference annotation dataset"), with Ensembl/UCSC/Other based chromosome identifiers, is to be used with a tool that also makes use of a different sourced reference genome or the reverse may be true, Ensembl/UCSC/Other sourced reference genome and a differnt sourced reference annotation.
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_adjust_identifiers_UCSC.md:10]:
```suggestion
- A GTF formatted dataset (potentially a "reference annotation dataset"), with Ensembl/UCSC/Other based chromosome identifiers, is to be used with a tool that also makes use of a different sourced reference genome or the reverse may be true, Ensembl/UCSC/Other sourced reference genome and a different sourced reference annotation.
```

**Review by beatrizserrano — COMMENTED** [review]
I don't understand this FAQ well. @jennaj or someone else, could you review it?

**Comment by jennaj** [comment]
Do the same as this one: https://github.com/galaxyproject/training-material/pull/3398

Then we drop this and the hub section points to Galaxy FAQs (top level). Should count as a "done" FAQ. Removing old content is important, too, not all will be migrated but the old hub FAQs can be cleaned up with a redirect.

**Comment by annefou** [comment]
This PR can be closed too. Thanks @Melkeb for your effort.

**Pull request closed without merging**